setup.py

```diff
@@ -15,6 +15,7 @@
 Programming Language :: Python :: 3.7
 Programming Language :: Python :: 3.8
 Programming Language :: Python :: 3.9
+Programming Language :: Python :: 3.10
```

Comment:
Is there anything that we do not support here? Can we put Python 3.10 to setup.py?

Reply:
We should include 3.10 in our tests:
https://github.com/MarketSquare/robotframework-robocop/blob/ae27292ba5cb0972dd983135f14ed7b7187df0af/.github/workflows/unit-tests.yml#L1

Previously we actually had issue with 3.9 due to some syntax I've used so it's good idea to have it in checks

Reply:
Updated. We'll see the results soon.

Reply:
Failed because 3.10 for yaml is 3.1 ;) Updated the code with '3.10'